types/react-bootstrap-typeahead/index.d.ts

```diff
@@ -0,0 +1,64 @@
+// Type definitions for react-bootstrap-typeahead 2.0
+// Project: https://github.com/ericgio/react-bootstrap-typeahead
+// Definitions by: My Self <https://github.com/Guymestef>
```

Comment:
Needs to fill in `My Self`, then should be good to merge

Reply:
Well I don't really mind to be flagged as the one who provided the typings but My Self replaced 😄